### PR TITLE
chore: Remove v1beta1 references

### PR DESCRIPTION
**Pull request by jigisha620**
Fixes #N/A 

**Description**
Dropping v1beta1 references for next release.

**How was this change tested?**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/apis/v1/taints.go [hunk 1]
```diff
@@ -42,7 +41,7 @@ var (
 	}
 )
 
-// IsDisruptingTaint checks if the taint is either the v1 or v1beta1 disruption taint.
+// IsDisruptingTaint checks if the taint is v1 disruption taint.
 func IsDisruptingTaint(taint v1.Taint) bool {
```
Comment (engedaam) [inline pkg/apis/v1/taints.go:45]: Do we still need this function?

File: hack/validation/labels.sh [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
-        {"message": "label domain \"karpenter.sh\" is restricted", "rule": "self.all(x, x in [\"karpenter.sh/capacity-type\", \"karpenter.sh/nodepool\"] || !x.find(\"^([^/]+)\").endsWith(\"karpenter.sh\"))"},
-        {"message": "label \"karpenter.sh/nodepool\" is restricted", "rule": "self.all(x, x != \"karpenter.sh/nodepool\")"},
-        {"message": "label \"kubernetes.io/hostname\" is restricted", "rule": "self.all(x, x != \"kubernetes.io/hostname\")"}]' -i pkg/apis/crds/karpenter.sh_nodepools.yaml 
-    # Vaild requirement value check
-    yqVersion="$Version" yq eval '.spec.versions[env(yqVersion)].schema.openAPIV3Schema.properties.spec.properties.template.properties.metadata.properties.labels.additionalProperties.maxLength = 63' -i pkg/apis/crds/karpenter.sh_nodepools.yaml
-    yqVersion="$Version" yq eval '.spec.versions[env(yqVersion)].schema.openAPIV3Schema.properties.spec.properties.template.properties.metadata.properties.labels.additionalProperties.pattern  = "^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?$"' -i pkg/apis/crds/karpenter.sh_nodepools.yaml
-done 
\ No newline at end of file
+# Adding validation to v1 APIs
+# Version = 0 // v1 API
```
Comment (jonathan-innis) [inline hack/validation/labels.sh:4]: nit: do you need these comments anymore for mapping the value in an array to a version

**Review by jonathan-innis — APPROVED** [review]
/lgtm
/approve

**Pull request merged**